File: pessoa.java
package java;

public class Pessoa {
    
    String nome;
    int idade;

    public Pessoa(String nome, int idade){
        this.nome=nome;
        this.idade=idade;
    }
    public void apresentar(){
        System.out.println("Nome:"+nome+"|idade:"+ idade);
    }
}

File: pessoa.js
class pessoa{
    constructor (nome, idade){
        this.nome= nome;
        this.idade= idade;
    }

    apresentar(){
        console.log(`nome:${this.nome} | idade:${this.idade}`);
    }
}

const pessoa=new pessoa("João,20");
pessoa.apresentar();
